data/brands/amenity/recycling.json

```diff
@@ -259,6 +259,22 @@
         "short_name": "SVdP"
       }
     },
+    {
+      "displayName": "Sympany",
+      "id": "sympany-dfbbb3",
+      "locationSet": {"include": ["nl"]},
+      "tags": {
+        "amenity": "recycling",
+        "brand": "Sympany",
+        "brand:wikidata": "Q2646891",
+        "name": "Sympany",
+        "operator": "Sympany",
```

Comment:
I notice you move this entry from operator tree to brand tree, I don't means I request a change, but just don't know which is better because I'm not a local mapper.

You can see those 2 comments that clarify which is brand and which is operator:
+ https://github.com/osmlab/name-suggestion-index/issues/5625#issuecomment-958218478
+ https://github.com/osmlab/name-suggestion-index/pull/5924#issuecomment-999635804

I have no problem with other entry.

Reply:
I've moved it back to operators, as that's probably the most import value, but still with some more details